x-pack/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/backfill/schedule.ts

```diff
@@ -759,7 +759,7 @@ export default function scheduleBackfillTests({ getService }: FtrProviderContext
         });
 
         it('should handle schedule request where some requests succeed and some requests fail appropriately', async () => {
-          const start = moment().utc().startOf('day').subtract(7, 'days').toISOString();
+          const start = moment().utc().startOf('day').subtract(14, 'days').toISOString();
```

Comment:
increasing the length of the backfill run because occasionally, checking AAD at the end of the this test would return a saved object not found error.